Contents/mods/StatsAPI/media/lua/client/StatsAPI/StatsAPI.lua

```diff
@@ -124,19 +124,38 @@ StatsAPI.addStressChangeDaily = function(sourceName, dailyChange)
     StatsAPI.Stress.modChanges[sourceName] = dailyChange / 86400 / 100
 end
 
----Adds a constant change to stress
+---Same as above, but calculated over an hour for higher precision
```

Comment:
These comments are intended to be read in the mouse-over tooltips, so 'same as above' should be avoided

Reply:
Fixed with current PR for stress